src/verkle_variants/rocksdb_prelagrange.rs

```diff
@@ -0,0 +1,70 @@
+use verkle_db::{
+    BareMetalDiskDb, BareMetalKVDb,
+    BatchDB, BatchWriter
+};
+use verkle_trie::{
+    Trie,
+    config::VerkleConfig,
+    database::VerkleDb,
+    committer::precompute::PrecomputeLagrange,
+};
+use crate::verkle_variants::traits::FFI;
+use rocksdb::{DB, WriteBatch};
+
+pub struct RocksDb(DB);
+pub struct _WriteBatch(WriteBatch);
+
+impl BareMetalDiskDb for RocksDb {
```

Comment:
what is the need of this, isnt this already done in the rust verkle repo?

Reply:
same for BareMetalDiskDb, BatchWriter, BatchDB, BareMetalKVDb?

Reply:
I tried to use the one defined in their repo but it didn't worked. I believe it's there for reference purpose.

Reply:
https://github.com/realnimish/rust-verkle-wrapper/pull/6